### PR TITLE
Clarify the method of selecting several lines for commenting in PR.

### PR DESCRIPTION
**Pull request by Denis-GH**
### Why:

When I read [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#:~:text=To%20add%20a%20comment%20on%20multiple%20lines%2C%20click%20and%20drag%20to%20select%20the%20range%20of%20lines%2C%20then%20click%20the%20blue%20comment%20icon.), I didn't understand where I needed to click to add a comment to multiple lines. So I suggest clarifying this.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Instead of just "click and drag" you should say where to click: "click on a line in the line number gutter area". In addition, there is another way to add a comment to several lines: "click on the blue comment icon and drag to select the range of lines".

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@Denis-GH Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by subatoi — CHANGES_REQUESTED** [review]
Thanks @Denis-GH, we appreciate you taking the time to raise a PR. Generally, we'd kindly ask if you could open an issue first in future, just so we can agree on an approach before going ahead with a solution.

In this case, I'm not confident all users will know what "gutter area" means, and it probably wouldn't be translated correctly. We can also take the opportunity to use an octicon here (`{% octicon "plus" aria-label="The plus sign icon" %}`) so I'd suggest this alternative solution:

1. Remove all the text after "To add a comment on multiple lines..." in the numbered list item in question.

1. Add a _new_ numbered list item as follows:

> 1. Optionally, you can add a comment on multiple lines. You can click the line number of the first line you want to comment on and drag down to select a range of lines, then click {% octicon "plus" aria-label="The plus sign icon" %} on the last line you want to comment on. Alternatively, you can click {% octicon "plus" aria-label="The plus sign icon" %} next to the first line you want to comment on, then drag down to the last line you want to comment on.

If you're happy to make those changes, then I'm happy to approve this and get it merged.

**Comment by Denis-GH** [comment]
@subatoi
Thak you for reply!
I have made the changes as per your suggestion.
I have also added this to another doc about this.

File: data/reusables/repositories/multiple-lines-comment.md [hunk 1]
```diff
@@ -0,0 +1 @@
+1. Optionally, you can add a comment on multiple lines. You can click the line number of the first line you want to comment on and drag down to select a range of lines, then click {% octicon "plus" aria-label="The plus sign icon" %} on the last line you want to comment on. Alternatively, you can click {% octicon "plus" aria-label="The plus sign icon" %} next to the first line you want to comment on, then drag down to the last line you want to comment on.
```
Comment (subatoi) [inline data/reusables/repositories/multiple-lines-comment.md:1]:
```suggestion
1. Optionally, you can add a comment on multiple lines. You can click the line number of the first line you want to comment on and drag down to select a range of lines, then click the blue comment icon on the last line you want to comment on. Alternatively, you can click the blue comment icon next to the first line you want to comment on, then drag down to the last line you want to comment on.
```

**Review by subatoi — DISMISSED** [review]
Many thanks @Denis-GH, and thank you for thinking about this so carefully with the reusables!

I have to apologise as I'm going to contradict something I said earlier, but I've realised we don't use an octicon for the same icon elsewhere in this same document, and instead we just refer to the "blue comment icon". For the sake of consistency, I've made a suggestion to change this back.

As soon as we're happy with that last point, I'll go ahead and merge this 👍

**Comment by subatoi** [comment]
I'll go ahead and merge this shortly :)

**Review by nguyenalex836 — APPROVED** [review]

**Comment by Candymanspoken** [comment]
> ### Why:
> When I read [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#:~:text=To%20add%20a%20comment%20on%20multiple%20lines%2C%20click%20and%20drag%20to%20select%20the%20range%20of%20lines%2C%20then%20click%20the%20blue%20comment%20icon.), I didn't understand where I needed to click to add a comment to multiple lines. So I suggest clarifying this.
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> Instead of just "click and drag" you should say where to click: "click on a line in the line number gutter area". In addition, there is another way to add a comment to several lines: "click on the blue comment icon and drag to select the range of lines".
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).
>   
>   * For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
> * [x]  For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

**Pull request merged**